tools/ratbagctl.in

```diff
@@ -1540,21 +1540,28 @@ class RatbagParserRoot(object):
         self.parser.print_help()
 
 
+def open_ratbagd():
+    try:
+        r = Ratbagd()
+    except RatbagdDBusUnavailable:
+        print("Unable to connect to ratbagd over dbus")
+        return None
+    else:
+        if not r.devices:
+            print("No devices available.")
+
+        return r
+
```

Comment:
just add an extra spare line here, I'm planning to fix ratbagctl for flake8, this will complain. Might as well fix it now already